ServerRoomMonitoring.Generator/Models/TemperatureSensor.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace ServerRoomMonitoringGenerator.Models
+{
+    public class TemperatureSensor
+    {
+        public int Temperature { get; set; }
+        public string Unit { get; set; }
+
+        public void generateValues()
+        {
+            Random random = new Random();
+            Temperature = random.Next(-20,40);
+            Unit = "Celcius";
+        }
+        public override string ToString()
+        {
+            return $"Temperature: {Temperature}, Unit: {Unit}";
```

Comment:
Wygodniej by było użyć parsowania na JSON i z powrotem po stronie serwera niż robić to ręcznie. Polecam tę bilbiotekę: https://www.newtonsoft.com/json. 
Jakbyśmy gadali o pozostałych sprawach, to fajnie by było po prostu stworzyć klasę DTO będącą kontraktem między serwerm i generatorem, która by była po prostu serializowana do JSONa w kanale. Jak potrzebne więcej wyjaśnień to słownie będzie łatwiej wytłumaczyć.